engines/bastion_katello/app/assets/javascripts/bastion_katello/products/views/products.html

```diff
@@ -9,7 +9,7 @@
   <div data-block="messages">
     <div bst-alerts success-messages="successMessages" error-messages="errorMessages"></div>
     <div bst-alert="success" ng-hide="productDeletionTaskId === undefined">
-      <button type="button" class="close" ng-click="unsetProductDeletionTaskId()">×</button>
+      <button type="button" class="close" ng-click="unsetProductDeletionTaskId()">x</button>
```

Comment:
I prefer the font awesome icon or `&times;`:

http://getbootstrap.com/css/#helper-classes-close

I think preference to the latter because it would probably be better for accessibility.